### PR TITLE
Add a common function WinCompilePackgeTest()

### PR DESCRIPTION
**Pull request by Francis-Liu**
The function **WinCompilePackgeTest** abstract common code from `.jenkins/Packaging.Jenkinsfile` and `.jenkins/Jenkinsfile`. Reference to this PR: https://github.com/openenclave/openenclave/pull/2531

### CODE REVIEW

File: src/jenkins/common/Openenclave.groovy [hunk 1]
```diff
@@ -120,4 +120,32 @@ def emailJobStatus(String status) {
     )
 }
 
-return this
\ No newline at end of file
+def WinCompilePackgeTest(String dirName, String buildType, String hasQuoteProvider, Integer timeoutSeconds) {
```
Comment (BRMcLaren) [inline src/jenkins/common/Openenclave.groovy]: WinCompilePackgeTest - > WinCompilePackageTest

Comment (jhand2) [inline src/jenkins/common/Openenclave.groovy]: I agree this is a good thing to add. I don't quite understand the name though. Is this function supposed to "compile package & run tests". At the very least a doxygen-style comment for the function would be good.

Comment (Francis-Liu) [reply src/jenkins/common/Openenclave.groovy]: Comment added.

Comment (Francis-Liu) [reply src/jenkins/common/Openenclave.groovy]: Fixed.

**Review by BRMcLaren — APPROVED** [review]

**Review by jhand2 — APPROVED** [review]

**Review by radhikaj — APPROVED** [review]

**Pull request merged**